server/utils/db/db_utils.py

```
@@ -1461,6 +1461,40 @@ def initialize_tables():
+            # the session tainted instead of hard-failing; every subsequent
+            # tool call then requires user approval via the command gate.
+            try:
+                cursor.execute("""
+                    ALTER TABLE chat_sessions
+                    ADD COLUMN IF NOT EXISTS security_tainted BOOLEAN NOT NULL DEFAULT false;
+                """)
+                logging.info(
+                    "Added security_tainted column to chat_sessions table (if not exists)."
+                )
+                conn.commit()
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)